source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Alter Down/123_DML_PIMS_FORM_TYPE_Alter_Down.sql

```diff
@@ -0,0 +1,48 @@
+/* -----------------------------------------------------------------------------
+Alter the data in the PIMS_FORM_TYPE table.
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2024-Jun-06  Initial version
+----------------------------------------------------------------------------- */
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Delete the "H1005" type
+DECLARE @CurrCd NVARCHAR(20)
+SET     @CurrCd = N'H1005'
+
+SELECT FORM_TYPE_CODE
+FROM   PIMS_FORM_TYPE
+WHERE  FORM_TYPE_CODE = @CurrCd;
+
+IF @@ROWCOUNT = 1
+  BEGIN
+  DELETE
```

Comment:
Hmm, I feel like this will crash if a document has been uploaded with the H1005 form type code - probably need to clear that first before executing this.

Reply:
the script file is duplicated I'll delete it.